src/export.js

```diff
@@ -1,18 +1,18 @@
-// Deprecated
-// Extend assert methods to QUnit for Backwards compatibility
-( function() {
-	var i,
-		assertions = Assert.prototype;
+function applyDeprecated( name ) {
+	return function() {
+		throw new Error(
+			name + " is now removed from QUnit 2.0, check out our upgrade guide at " +
```

Comment:
I think all of the new errors should identify context (e.g., global {name} vs. QUnit.{name}), but especially here where it's not obvious.